docs/guide/module-hierarchy.rst

```diff
@@ -273,7 +317,15 @@ Integration tests for ``geotrellis-spark``.
 geotrellis-store
 ----------------
 
-TODO: Grisha complete this section
+In GeoTrellis 3.0 it was decided to call ``backends`` where the data is stored (i.e. S3, HDFS, Accumulo, etc.) to call ``stores``.
```

Comment:
A bit of a nitpick, but shouldn't we describe here how things exist at the present time? How things relate to old things that might have once been would better belong in a changelog, right? Perhaps we could split the difference with something like:
```
Types and interfaces for interacting with a number of different storage backends in an abstract way.

In older versions of GeoTrellis, ``store`` implementations were referred to as ``backends``. 

*Provides:* ``geotrellis.store.*``

- Includes concrete reference implementations for the ``File`` and ``Hadoop`` stores.
- <... include rest of the list you wrote up here>
```